docs/en/Deployment.md

```diff
@@ -0,0 +1,145 @@
+ 
+### Then start the second Master, on machine B, which also configures NameServer's IP: 192.168.1.1.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-noslave/broker-b.properties &
+
+...
+```
+
+The boot command shown above is used in the case of a single NameServer.For clusters of multiple NameServer, the address list after the -n argument in the Broker boot command is separated by semicolons, for example, 192.168.1.1: 9876;192.161.2: 9876.
+
+### 3 Multiple Master And Multiple Slave Mode-Asynchronous replication
+
+Each Master configures a Slave, with multiple pairs of Master-Slave.HA uses asynchronous replication, with a short message delay (millisecond) between Master and Slave.The advantages and disadvantages of this mode are as follows:
+
+- Advantages: even if the disk is corrupted, very few messages will be lost and the real-time performance of the message will not be affected;At the same time, when Master is down, consumers can still consume messages from Slave, and the process is transparent to the application itself and does not require human intervention;Performance is almost as high as multiple Master mode;
+- Disadvantages:a small number of messages will be lost when Master is down or the disk is corrupted.
+
+The starting steps for multiple Master and multiple Slave mode are as follows:
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+**2）Start the Broker cluster**
+
+```shell
+### For example, starting the first Master, on machine A assumes that the IP for configuring NameServer is: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-a.properties &
+ 
+### Then start the second Master, on machine B, which also configures NameServer's IP: 192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-b.properties &
+ 
+### Then start the first Slave,on machine C assumes that the IP for configuring NameServer is:192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-a-s.properties &
+ 
+### Last start the second Slave,on machine D assumes that the IP for configuring NameServer is:192.168.1.1 and port is 9876.
+$ nohup sh mqbroker -n 192.168.1.1:9876 -c $ROCKETMQ_HOME/conf/2m-2s-async/broker-b-s.properties &
+```
+
+The above shows a startup command for 2M-2S-Async mode, similar to other nM-nS-Async modes.
+
+### 4 Multiple Master And Multiple Slave Mode-Synchronous dual write
+
+In this mode, one Slave is also configured for each Master and there are multiple pairs of Master-Slave.HA uses synchronous double-write, that is, only Master and Slaver write successfully to return success to the application.
```

Comment:
"that is, the success response will be returned to the application only when the message is successfully written into the master node and replicated to more than one slave node."  seems better.